content/en/docs/tutorials/kubernetes-basics/deploy-app/deploy-interactive.html

```diff
@@ -17,6 +17,14 @@
 
     <main class="content katacoda-content">
 
+        <div class="row">
+            <div class="col-md-12">
+                <p>
+                    A Pod is the basic execution unit of a Kubernetes application–the smallest and simplest unit in the Kubernetes object model that you create or deploy. A Pod represents processes running on your Cluster. <a href="/docs/concepts/workloads/pods/pod-overview/#understanding-pods">Learn more about pods</a>
```

Comment:
This relative link didn't actually work in my testing.(Testing with a local Hugo environment) 

But I see similar relative urls elsewhere in the docs so I assumed an absolute url is not the way to go.

```suggestion
                    A Pod is the basic execution unit of a Kubernetes application–the smallest and simplest unit in the Kubernetes object model that you create or deploy. A Pod represents processes running on your Cluster. <a href="https://kubernetes.io/docs/concepts/workloads/pods/pod-overview/#understanding-pods">Learn more about pods</a>
```

Reply:
Error with above relative url:
![image](https://user-images.githubusercontent.com/15719653/75826620-c684da00-5d9f-11ea-90d4-37c1fa27a09e.png)

Reply:
Ah nice, it works in staging environment. 😄